bin/execute_and_publish_performance_test.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+current_time=$(date "+%Y.%m.%d-%H.%M.%S")
+perf_test_aws_s3_bucket=${PERF_TEST_AWS_S3_BUCKET:-notify-perfortmance-test-results-staging}
```

Comment:
Small typo here in `perfortmance` (the extra `t`).

Reply:
Done